odxtools/companydata.py

```diff
@@ -90,7 +90,8 @@ def read_xdoc_from_odx(xdoc):
                 position=position,
                 )
 
-def read_company_datas_from_odx(et_element, doc_frags: List[OdxDocFragment]):
+def read_company_datas_from_odx(et_element, doc_frags: List[OdxDocFragment]) \
+        -> Optional[NamedItemList[CompanyData]]:
```

Comment:
I think it would be better to return an empty list rather than return none

Reply:
same as above: we must be able to distinguish "not specified" and "empty list" somehow.

Reply:
yes, but the `COMPANY-DATAS` tag is optional:
```xsd
            <xsd:element type="COMPANY-DATAS" name="COMPANY-DATAS" minOccurs="0" maxOccurs="1"/>
```

anyway, I this can be distinguished by checking if the list is empty or not. fixed, thanks for the persistence. (there are still plenty of other occurences where this happens, though...)